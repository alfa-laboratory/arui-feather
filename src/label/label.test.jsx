/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from 'react';
import { mount } from 'enzyme';
import Label from './label';

describe('label', () => {
    it('should render without problems', () => {
        let label = mount(<Label>Label-test</Label>);

        expect(label).toMatchSnapshot();
    });

    it('should render with size=`m` by default', () => {
        let label = mount(<Label />);

        expect(label.getDOMNode().className).toContain('label_size_m');
    });
});
