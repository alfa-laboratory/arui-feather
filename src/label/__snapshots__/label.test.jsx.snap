// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`label should render without problems 1`] = `
<Label
  isNoWrap={false}
  size="m"
>
  <span
    className="label label_size_m label_theme_alfa-on-white"
  >
    <span
      className="label__inner"
    >
      Label-test
    </span>
  </span>
</Label>
`;
